meta-xilinx-bsp/recipes-bsp/u-boot/u-boot-xlnx_2019.2.bb

```diff
@@ -9,6 +9,8 @@ include u-boot-spl-zynq-init.inc
 
 SRC_URI_append_kc705-microblazeel = " file://microblaze-kc705-Convert-microblaze-generic-to-k.patch"
 
+SRC_URI += "file://0003-arm64-zynqmp-configs-Fix-sdbootdev-environment-varia.patch" 
```

Comment:
instead of the patch, based on a flag we can define the sdbootdev in the boot script for u-boot-zynq-scr. This would prevent patching the u-boot. I normally dont hold patches that cannot be upstreamed

Reply:
It will not work for u-boot-zynq-uenv flow then, as [u-boot](https://github.com/Xilinx/u-boot-xlnx/blob/xilinx-v2019.2/include/configs/xilinx_zynqmp.h) provides:
`
sd_uEnvtxt_existence_test=test -e mmc $sdbootdev:$partid /uEnv.txt
`
where wrong _sdbootdev_ is used before the script is loaded.

Reply:
i agree, but why use both scr and uEnv.txt? with distroboot in u-boot, just using u-boot-scr should be sufficient.

Reply:
It's fine with me. I just wanted to keep compatibility with the obsolete uEnv flow. In this case, I can implement the proposed changes only for u-boot-scr.

Reply:
Since the default is distroboot, i recommend we stick with it . I am planning to send a patch to master to remove uEnv.txt creation after zeus is cut

Reply:
I have just committed the fix.